R/proj.R

```diff
@@ -287,7 +287,7 @@ project_pkgdown_url <- function(base_path = proj_get()) {
 #' @export
 proj_activate <- function(path) {
   check_path_is_directory(path)
-  path <- user_path_prep(path)
+  path <- proj_path_prep(path)
```

Comment:
This makes me flinch. There's a really strong policy, with good reason, for always processing a user-supplied path with `user_path_prep()`, whereas `proj_path_prep()` is an internal function with a different purpose.

I am studying this now, but suspect this can't be the right solution.

Reply:
https://github.com/r-lib/usethis/blame/master/principles.md#L48

Reply:
Ah sorry about that. If you have suggestions, I'm happy to update. Otherwise, I can take a look again

Reply:
I think I have a (different) solution. But now I need to test it.